src/main/bindings/BindingsPlugin.ts

```diff
@@ -33,4 +42,35 @@ export abstract class BindingsPlugin  {
      * @param graphs
      */
     abstract async export(configuration: ConfigurationParameter[], graphs: meta.DialectWrapper[]): Promise<Resource[]>
+
+    abstract updateBindings(bindName : string): void
+    abstract initBindings(bindUuid: string): string
+
+    private bindName = namedNode(schPref.bind + 'Binding')
+    private uuidName = namedNode(schPref.amldata + 'uuid')
+    private bdsName = namedNode(schPref.bind + 'bindingDeclarationSource')
+    private bsName = namedNode(schPref.bind + 'bindingSource')
+    private bbName = namedNode(schPref.bind + 'boundBy')
+    private bName = namedNode(schPref.bind + 'binding')
+    protected dde = namedNode('http://a.ml/vocabularies/meta#DialectDomainElement')
+    protected de = namedNode('http://a.ml/vocabularies/document#DomainElement')
+
+    createBinding(regime: string, bindingDeclSrc: string, source : string){
+        let uuid = uuidv4()
+        let bindName = 'http://mulesoft.com/modeling/bindings/' + uuid
+        let bindNode = namedNode(bindName)
+        let sourceNode = namedNode('http://mulesoft.com/modeling/instances/uuid/'+source)
+        let graph = namedNode(regime)
+        let bindingDeclarationSource = namedNode(bindingDeclSrc)
+        let stupid = `file://${process.cwd()}/node_modules/@api-modeling/api-modeling-metadata/model/bindings/schema/modelBindingsDialect.yaml#/declarations/Binding`
+        store.addQuad(bindNode, rdfType, bindName,graph)
```

Comment:
Why are we adding this to the store? If you generate bindings, add them to a bindings model